tech/languages/python/python-installation.md

```diff
@@ -57,17 +72,22 @@ $ source project_venv/bin/activate
 When the virtual environment is activated (you can see its name in brackets at the beginning of your prompt), you can install modules via `pip install`.
 
 ```bash
-(project_venv) $ pip install name_of_module
+(project_venv) $ python -m pip install requests
 ```
 
 That is all, you have successfully created your own virtual environment. Now you can run Python (see above) and start working on your project.
 
-When you finish your work, just deactivate the virtual environment.
+When you finish your work, you can deactivate the virtual environment.
 
 ```bash
 (project_venv) $ deactivate
 ```
 
+Note that packages installed system-wide are not available in the virtual environments:
```

Comment:
Probably a typo `:`.

Reply:
It's intentional, the sentence continues on the next line.